.github/workflows/main.yml

```diff
@@ -140,7 +140,7 @@ jobs:
       - name: Install prerequisites
         run: |
           python3 -m pip install -r requirements.txt
-          python3 -m pip install twine
+          python3 -m pip install twine wheel
```

Comment:
I've seen that before, probably makes sense.

Why did you remove `twine`? I believe that's required to publish the release, no?

Reply:
I did not, it's right before `wheel` :laughing:. What time was it when you read that? :grin:

Reply:
Gosh. Thanks.